src/ServiceProvider.php

```diff
@@ -47,8 +59,19 @@ private function configureCommands()
     {
         if ($this->app->runningInConsole()) {
             $this->commands([
-                Init::class, Sync::class
+                Init::class,
+                Sync::class,
+                SyncAndPurge::class,
+                SyncAndShowPurgeable::class
             ]);
         }
     }
+
+    /**
+     * Include the gettext functions
+     */
+    public static function improveGettextFunctions()
+    {
+        include_once __DIR__.'/translator_functions.php';
```

Comment:
We can instruct composer to autoload helpers for us, see example here:
https://github.com/armandsar/laravel-quick-view/blob/master/composer.json#L30